diagnostic.md

```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    Tell the program what data to pull and what view to see given a specific URL
```

Comment:
That is what a Route does.
The Router is where we specify route definitions.

.5/1